amplify/backend/function/wandb/src/utils/upload.py

```diff
@@ -9,4 +9,8 @@ def upload(path):
         Filename=path,
         Bucket=os.environ.get("STORAGE_S359BCE836_BUCKETNAME"),
         Key="public/multi.json", # Ref: https://docs.amplify.aws/javascript/build-a-backend/storage/configure-access/
+        # Ref: https://docs.amplify.aws/javascript/build-a-backend/storage/download/#frequently-asked-questions
+        # Ref: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control
+        # using the no-cache directive indicates that response can be cached but before reusing it, validate it from origin server
+        ExtraArgs={"Metadata": {"Cache-Control": "no-cache"}}

```

Comment:
This header was ovewrritten by amazon as `X-Amz-Meta-Cache-Control:` automatically